docs/reference/developer/getting-started.rst

```diff
@@ -101,6 +101,34 @@ In Python:
 
   import scipp as sc
 
+Instead of ``conda develop``, some developers may prefer using ``pip`` to make an editable install.
+Scipp uses ``scikit-build``, which currently does not fully support this directly.
+Therefore, we need to call ``cmake`` manually in this case and install into the Python source directory, or create symlinks.
+We have configured ``tox`` for this purpose:
+
+.. code-block:: bash
+
+  cmake --preset base -DCONAN_TBB=ON
+  cmake --build --preset build
+  tox -e editable
+  pip install -e .
```

Comment:
My previous comment still stands. Instead of this line, you can also use `conda develop src`

Reply:
But will `pip` know about this?

Reply:
No. But why do you need it to?

Reply:
I would never mix the two. Either install all local packages with pip or with conda.

Reply:
See update.